Makefile

```diff
@@ -55,6 +55,25 @@ $(OUT)/emulate.o: CFLAGS += -fno-gcse -fno-crossjumping
 endif
 endif
 
+MINI_GDBSTUB :=
```

Comment:
Replace `MINI_GDBSTUB` with `LIBGDBSTUB` since we might integrate the whole repository later.